packages/suite/src/support/messages.ts

```diff
@@ -2892,6 +2892,35 @@ export default defineMessages({
         defaultMessage: 'System',
         id: 'TR_SETTINGS_SAME_AS_SYSTEM',
     },
+    TR_SETTINGS_DEVICE_BANNER_TITLE_DISCONNECTED: {
+        defaultMessage: 'Connect device to access device settings',
+        id: 'TR_SETTINGS_DEVICE_BANNER_TITLE_DISCONNECTED',
+    },
+    TR_SETTINGS_DEVICE_BANNER_TITLE_BOOTLOADER: {
+        defaultMessage: 'Other settings unavailable in bootloader mode',
+        id: 'TR_SETTINGS_DEVICE_BANNER_TITLE_BOOTLOADER',
+    },
+    TR_SETTINGS_DEVICE_BANNER_TITLE_REMEMBERED: {
+        defaultMessage: 'Connect device to change device settings',
+        id: 'TR_SETTINGS_DEVICE_BANNER_TITLE_REMEMBERED',
+    },
+    TR_SETTINGS_DEVICE_BANNER_DESCRIPTION_DISCONNECTED: {
+        defaultMessage: 'TODO',
+        id: 'TR_SETTINGS_DEVICE_BANNER_DESCRIPTION_DISCONNECTED',
+    },
```

Comment:
`TODO`, not used

Reply:
https://github.com/trezor/trezor-suite/pull/4664/commits/559bc0401adf2d624a64f571bfe072ce8f346bc2